task/t2.py

```diff
@@ -0,0 +1,25 @@
+from utils.read_files import read_files_t2
+
+file_values = read_files_t2()
+
+temperatures = []
+for file_value in file_values:
+    if file_value and len(file_value) >= 4 and file_value[1] and file_value[3]:
+        date = file_value[0]
+        min_temp = int(file_value[3])
+        max_temp = int(file_value[1])
+        temperatures.append({'date': date, 'min_temp': min_temp, 'max_temp': max_temp})
+
+# Filter dates with a difference of exactly 7 between min and max temp
+filtered_temperatures = []
+for temp in temperatures:
+    if abs(temp['max_temp'] - temp['min_temp']) == 7:
```

Comment:
what does abs() do? and why are you using that even? trying to be fancy?

Reply:
abs() is a built-in Python function that returns the absolute value of a number. The absolute value of a number is its distance from zero, regardless of its sign.